README.md

```diff
@@ -72,11 +72,22 @@ end
 
 To install the latest release as a gem:
 
-`gem install rack-test`
+```
```

Comment:
This modification is that I wanted to show same color for the section of `Install as a gem` and `Install via Bundler`.
`<code>..</code>`, `<pre>...</pre>` are displayed as different color on github markdown.